bedrock/firefox/templates/firefox/home/index-master.html

```diff
@@ -0,0 +1,284 @@
+  {% set get_amazon = 'Amazon' %}
+  {% set product_browsers = _('Browsers') %}
+  {% set product_monitor = 'Monitor' %}
+  {% set product_send = 'Send' %}
+  {% set product_lockwise = 'Lockwise' %}
+  {% set product_pocket = 'Pocket' %}
+{% endif %}
+
+{% block content %}
+<main role="main">
+
+    {% call hero(
+      title=page_title,
+      class='mzp-has-image',
+      heading_level=1,
+      image_url='firefox/home/master/hero.jpg',
+      include_highres_image=true
+    ) %}
+    {% endcall %}
+
+    <div class="c-showcase">
+      <div class="mzp-l-content">
+        <h2 class="c-showcase-title">{{ family_title }}</h2>
+        <ul class="c-showcase-list">
+          <li>
+            <a class="mzp-c-cta-link" href="{{ url('firefox.new') }}" data-cta-type="link" data-cta-text="Browsers">
+              <img alt="" src="{{ static('img/logos/firefox/quantum/logo-md.png') }}" height="40" width="40"><br>
+              {{ product_browsers }}
+            </a>
+          </li>
+          <li>
+            <a class="mzp-c-cta-link" href="https://monitor.firefox.com{{ referrals }}" rel="external noopener" data-cta-type="link" data-cta-text="Monitor">
+              <img alt="" src="{{ static('/protocol/img/logos/firefox/monitor/logo-md.png') }}" height="40" width="40"><br>
+              {{ product_monitor }}
+            </a>
+          </li>
+          <li>
+            <a class="mzp-c-cta-link" href="https://send.firefox.com/{{ referrals }}" rel="external noopener" data-cta-type="link" data-cta-text="Send">
+              <img alt="" src="{{ static('/protocol/img/logos/firefox/send/logo-md.png') }}" height="40" width="40"><br>
+              {{ product_send }}
+            </a>
+          </li>
+          <li>
+              <a class="mzp-c-cta-link" href="https://lockwise.firefox.com/{{ referrals }}" rel="external noopener" data-cta-type="link" data-cta-text="Lockwise">
+                <img alt="" src="{{ static('/protocol/img/logos/firefox/lockwise/logo-md.png') }}" height="40" width="40"><br>
+                {{ product_lockwise }}
+              </a>
+            </li>
+          <li>
+            <a class="mzp-c-cta-link" href="https://getpocket.com{{ referrals }}" rel="external noopener" data-cta-type="link" data-cta-text="Pocket">
+              <img alt="" src="{{ static('img/logos/pocket/logo.svg') }}" height="40" width="40"><br>
+              {{ product_pocket }}
+            </a>
+          </li>
+        </ul>
+      </div>
+    </div>
+
+    <div class="mzp-l-content">
+      {% call feature_card(
+        title=browsers_trackers,
+        heading_level=3,
+        image_url='firefox/home/master/browser.png',
+        include_highres_image=true,
+        class='mzp-l-card-feature-left-half t-browsers',
+        ga_title='Trackers',
+        media_after=true
+      ) %}
+
+        <div class="mzp-c-menu-list mzp-t-cta mzp-t-download">
+          <h4 class="mzp-c-menu-list-title">{{ get_browser }}</h4>
```

Comment:
From an SEO perspective I think "Firefox Browser" should probably be the heading here, but the wordmark is currently a background image, which search engines can't see? Same applies for the other products.

Reply:
I copied what you did on the privacy pages.